### ISSUE TITLE
Failed to get registry value: More data is avaiable.

### ISSUE THREAD

**Issue by PolGraphic** [nitsch/psmove-pair-win#4]
According to https://github.com/thp/psmoveapi/issues/172

After downloading and using `psmovepair` followed by `psmove-pair-win` I got:
![2](https://cloud.githubusercontent.com/assets/2772514/7480317/11d1f774-f368-11e4-9a66-5025e96301d6.jpg)

The entry in registry looked like that:

![3](https://cloud.githubusercontent.com/assets/2772514/7480333/383b74a8-f368-11e4-9a1d-28d8fef58935.PNG)

As cboulay suggested, I had to change the `VirtuallyCabled` to `1` manually (with `regedit`).

**Comment by cboulay** [comment]
Additional piece of information: They were using a version of psmove-pair-win.exe that I compiled from @betonme's pull request #2

**Comment by betonme** [comment]
Hi,

thanks for the good report.

I guess the problem is caused by the leading zeros in the calculation of the registry "path".

Update: That is not the problem.

**Comment by betonme** [comment]
Hi PolGraphic,

can You tell me the full name of the directory "e0cec39a...." within the Regirstry?

Have You started the psmove-pair-win.exe as administrator?

Thanks

**Comment by PolGraphic** [comment]
Of course, the full name of the directory is `e0cec39a12660006f71c9402`.
I've not started the `psmove-pair-win.exe` as administrator. But if it asked me for privileges (I don't remember now), I clicked "yes".

**Comment by betonme** [comment]
Are You using Windows 8.0, 8.1 32 or 64 Bit?

Can You test the changes from this commit?
https://github.com/betonme/psmove-pair-win/commit/564cc59cfa66f8dde661957b926ff5a4b7c3b15b

Thanks
